File: Practical Programs/WPT_LAB/Assignment 11/Employee.js
// Employee.js
class Employee {
  constructor(firstName, lastName, email, designation, basicSalary) {
    this.fname = firstName;
    this.lname = lastName;
    this.email = email;
    this.des = designation;
    this.sal = basicSalary;
  }

  display() {
    console.log("Employee Details are:");
    console.log(`Name: ${this.fname}, LastName: ${this.lname}, Email: ${this.email}, Designation: ${this.des}, BasicSalary: ${this.sal}`);
  }

  getFname() { return this.fname; }
  setFname(newName) { this.fname = newName; }

  getLname() { return this.lname; }
  setLname(newName) { this.lname = newName; }

  getEmail() { return this.email; }
  setEmail(newEmail) { this.email = newEmail; }

  getDesg() { return this.des; }
  setDesg(newDes) { this.des = newDes; }

  getSal() { return this.sal; }
  setSal(newSal) { this.sal = newSal; }
}

module.exports = Employee;

File: Practical Programs/MS .NET/SolutionCSharpDemos/EmpMgnt_WindowsForms/Employee.cs
namespace EmpMgnt_WindowsForms
{
    public class Employee
    {
        public string Name { get; }
        public string Email { get; }
        public string Designation { get; }

        public Employee(string name, string email, string designation)
        {
            Name = name;
            Email = email;
            Designation = designation;
        }

        public override string ToString()
        {
            return $"{Name} | {Email} | {Designation}";
        }
    }
}
